packages/graphql-migrations/src/plugin/SchemaTransformer.ts

```diff
@@ -15,12 +15,8 @@ export const removeNonModelTypesFromSchema: SchemaTransformerPlugin = {
     }
 
     const schemaComposer = new SchemaComposer(schema)
-    for (const userType of nonModelTypes) {
-      schemaComposer.delete('StringInput')
-    }
```

Comment:
@wtrocki @craicoverflow this was basically repeating the same operation, so no need to use the loop